src/storage/cuckoo/cuckoo.h

```diff
@@ -18,7 +18,7 @@
 #define CUCKOO_POLICY CUCKOO_POLICY_RANDOM
 #define CUCKOO_MAX_TTL (30 * 24 * 60 * 60) /* 30 days */
 #define CUCKOO_DATAPOOL NULL
-#define CUCKOO_PREFAULT false
+#define CUCKOO_PREFAULT true
```

Comment:
why?

Reply:
The intention was to use a function which was called before commit ec0ef42.
For cuckoo_setup method it was **cc_zalloc**, with this changes it is achieved when prefault option is set to true.
That's why I want it to have CUCKOO_PREFAULT set to true.

Reply:
Does anything rely on the memory being zeroed? If so, change the datapool_shm to call zalloc. If not, then keep this as false.

This is a debug/benchmarking config, and it shouldn't need to be set to make sure things work. On top of that, the name of the variable doesn't say anything about zeroing.